packages/core/test/foundry/optimistic-assertor/OptimisticAsserter.Events.t.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "./Common.sol";
+
+contract OptimisticAsserterEvents is Common {
+    function setUp() public {
+        OptimisticAssertorFixture.OptimisticAsserterContracts memory oaContracts =
+            new OptimisticAssertorFixture().setUp();
+        optimisticAssertor = oaContracts.optimisticAssertor;
+        defaultCurrency = oaContracts.defaultCurrency;
+        mockOracle = oaContracts.mockOracle;
+        timer = oaContracts.timer;
```

Comment:
can we use `_commonSetup();` here?
then we could avoid numerous calls on `optimisticAssertor.defaultBond()` and `optimisticAssertor.defaultLiveness()` and use stored state variables instead.